{"uuid": "76715b51-b4a9-432c-8db9-473f3c4d4c1a", "children": ["0199381a-c37a-42f1-9b99-b7bbc9e46d3f"], "befores": [{"name": "action", "status": "broken", "statusDetails": {"message": "AttributeError: module 'allure' has no attribute 'environment'\n", "trace": "  File \"/usr/local/Cellar/python/3.6.5/Frameworks/Python.framework/Versions/3.6/lib/python3.6/site-packages/pluggy/callers.py\", line 187, in _multicall\n    res = hook_impl.function(*args)\n  File \"/usr/local/Cellar/python/3.6.5/Frameworks/Python.framework/Versions/3.6/lib/python3.6/site-packages/_pytest/fixtures.py\", line 964, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n  File \"/usr/local/Cellar/python/3.6.5/Frameworks/Python.framework/Versions/3.6/lib/python3.6/site-packages/_pytest/fixtures.py\", line 792, in call_fixture_func\n    res = fixturefunc(**kwargs)\n  File \"/Users/test/work/API_Automation/TestCase/conftest.py\", line 35, in action\n    allure.environment(environment=env)\n"}, "start": 1586763318841, "stop": 1586763318842}], "start": 1586763318841, "stop": 1586763318846}